docs/reference/deploy/config/master-config-reference.rst

```diff
@@ -151,6 +151,12 @@ The default list of devices to pass to the Docker daemon. Ignored by resource ma
 The default bind mounts to pass to the Docker container. Ignored by resource managers of type
 ``kubernetes``. See :ref:`resources.devices <exp-bind-mounts>` for more details.
 
+``kubernetes``
+==============
+
+-  ``max_slots_per_pod`` See :ref:`resource_manager.max_slots
+   <master-config-reference-max-slots-per-pod>` for more details.
+
```

Comment:
does not need to be a bullet, that would make it different from the rest of the page

``kubernetes``
==============

``max_slots_per_pod`` See :ref:`resource_manager.max_slots
   <master-config-reference-max-slots-per-pod>` for more details.